src/cdrizzlebox.c

```diff
@@ -839,22 +968,44 @@ do_kernel_square(struct driz_param_t *p) {
                 w = 1.0;
             }
 
+	    /* Pre-compute slopes and sign of dx for each segment,
+	       since they will be used for all pixels in the loop.
+	       Also compute the inverse of the slope to avoid more
+	       division calls later.
+	     */
+
+	    for (ii = 0; ii < 4; ii++) {
+	        dx = xout[(ii+1) & 0x3] - xout[ii];
+	        dy = yout[(ii+1) & 0x3] - yout[ii];
+	        if (dx >= 0)
+		    sgn_dx[ii] = 1;
+	        else
+		    sgn_dx[ii] = -1;
+	        slope[ii] = dy / dx;
+	        inv_slope[ii] = dx / dy;
+	    }
+	    inv_jaco = 1.0 / jaco;
```

Comment:
Maybe just do this:
```suggestion
        inv_jaco = 2.0f / ((xout[1] - xout[3]) * (yout[0] - yout[2]) -
                           (xout[0] - xout[2]) * (yout[1] - yout[3]));
```
and remove `jaco` variable altogether?